hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el

```diff
@@ -0,0 +1,92 @@
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "2423918326 3522900596"
+INSTANCE: tb.dut.gen_partitions[4].gen_buffered.u_part_buf
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "2423918326 3522900596"
+INSTANCE: tb.dut.gen_partitions[5].gen_buffered.u_part_buf
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "2423918326 3522900596"
+INSTANCE: tb.dut.gen_partitions[6].gen_lifecycle.u_part_buf
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "3522900596"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+CHECKSUM: "1031540710 109269272"
+INSTANCE: tb.dut.gen_partitions[0].gen_unbuffered.u_part_unbuf
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->AccessError "6->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->AccessError "7->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: unreachable for unbuffered partition only"
+Transition AccessError->CheckFailError "5->6"
+CHECKSUM: "1031540710 109269272"
+INSTANCE: tb.dut.gen_partitions[1].gen_unbuffered.u_part_unbuf
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->AccessError "6->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition CheckFailError->FsmStateError "6->7"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->AccessError "7->5"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "109269272"
+ANNOTATION: "[UNR]: unreachable for unbuffered partition only"
+Transition AccessError->CheckFailError "5->6"
+CHECKSUM: "4056579122 2961818352"
+INSTANCE: tb.dut.u_otp_ctrl_dai
+Fsm error_q "2961818352"
+ANNOTATION: "[UNR]: this error state is a terminal state"
+Transition FsmStateError->AccessError "7->5"
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_sender is fully verified by FPV testbench."
+MODULE:prim_diff_decode
```

Comment:
this should be excluded if it's in prim_alert_sender as we exclude the entire prim_alert_sender, right?

Reply:
Thanks weicai for taking the time to discuss the issue with me offline. What we found is that:
1. We are not able to use `*` notations to wildcard exclude alert_sender instances.
2. If we exclude module `prim_alert_sender`, it does not exclude `prim_diff_decode` automatically even though `prim_diff_decode` is inside `alert_sender`.
3. For now we will do this for all IPs to unblock coverage requirements. Once #6089 is landed, maybe we will have a better solution.